tests/smoke-tests/defaults.test.ts

```diff
@@ -99,19 +99,18 @@ for (let packageManager of SUPPORTED_PACKAGE_MANAGERS) {
       expect(exitCode).toEqual(0);
     });
 
-    it('lint with fixtures pass', async () => {
+    it('build and test ', async () => {
+      // Copy over fixtures
       await helper.fixtures.use('./my-addon/src/components');
       await helper.fixtures.use('./test-app/tests');
 
+      // Ensure that we have no lint errors.
+      // It's important to keep this along with the tests, 
+      // so that we can have confidence that the lints aren't destructively changing
+      // the files in a way that would break consumers
       let { exitCode } = await helper.run('lint:fix');
 
       expect(exitCode).toEqual(0);
-    });
-
-    it('build and test ', async () => {
-      // Copy over fixtures
-      await helper.fixtures.use('./my-addon/src/components');
```

Comment:
re-copying over the fixtures here hid this problem from us in https://github.com/embroider-build/addon-blueprint/pull/159/